frontend/lib/src/components/core/Block/utils.ts

```diff
@@ -149,4 +149,10 @@ export interface BaseBlockProps {
    * from that callback.
    */
   formsData: FormsData
+
+  /**
+   * If true , the element should not allow going into fullscreen. Right now we plan
+   * to use it, for example, in Dialogs to prevent fullscreen issues.
+   */
+  hideFullScreenButton?: boolean
```

Comment:
nit: reading the docstring comment above, I'm questioning a bit whether `hideFullScreenButton` is the best name here. What we actually want is to disable the fullscreen mode for elements, and hiding the button is the best way to do this currently since it is the only way to open elements in fullscreen. Maybe we can "unify" this under the name: `disableFullscreenMode` as used in libConfig. Might make even more sense together with some refactoring of the `libConfig.disableFullscreenMode` usage as suggested above. 

Could be confusing to have a concept called: `disableFullscreenMode` and `hideFullScreenButton` with both basically doing the same. But that is something I messed up when I initially implemented that.

Reply:
Yeah that's a good point! I had similar thoughts when doing this. I will give merging this somehow another look 👍